frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/citizen/index.js

```
@@ -224,8 +583,53 @@ const Home = ({
             window.open(window?.globalConfigs?.getConfig?.("DIGIT_HOME_URL"), "_blank").focus();
           }}
         />
+      </div> */}
+       <div style={containerStyle}>
+      <div style={innerContainerStyle}>
+        <div style={columnStyle}>
+          <div style={titleStyle}>Term and Policies</div>
+          <a href="#" style={linkStyle}>Privacy Policy</a>
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)